client/components/EventNotFound.jsx

```diff
@@ -25,11 +29,13 @@ const EventNotFound = (props) => {
 EventNotFound.propTypes = {
   mainMessage: PropTypes.string,
   statusMessage: PropTypes.string,
+  actions: PropTypes.array,
 };
 
 EventNotFound.defaultProps = {
   mainMessage: '',
   statusMessage: '',
+  actions: null,
```

Comment:
Since PropType is array, I was thinking we could make this array too "[]"

Reply:
+1